### PR TITLE
refactor datos_info

### PR DESCRIPTION
**Pull request by natilou**
- Refactor of the function `datos_info()`, I divided every part of this function in different methods of a class `GenerateDataInfo()`
- I didn't modified any logic and I respected the order that every part of the function was executed. 
- Rename the function `get_phone_risk()` to `get_phones()` and add some arguments to use this function for `risk_data()` and for `GenerateDataInfo()`
- Added tests to  `GenerateDataInfo()`
- In another PR I will move `get_phones()` and `read_osiris_accounts()` functions to a helper file, because they are used by other methods. 
- All the methods of the class are `classmethods` because the class doesn't have to save a state, there is no need to have an instance of the class, but I think using a class makes easier to read and assign the responsibility for this use case.

### CODE REVIEW

File: src/risk_data.py [hunk 1]
```diff
@@ -53,7 +57,7 @@ def risk_data():
 
     df_risk = pd.merge(uploaded_accounts, df_risk, how="inner", on="DNI")
 
-    df_phone_risk = get_phone_risk(df_risk=df_risk)
+    df_phone_risk = get_phones(df=df_risk)
```
Comment (matiasbertani) [inline src/risk_data.py:60]: If we redefine this function to `get_phones(df: pd.DataFrame, stop: int, colum_name: str)`. Shoukld we pass the `stop` and `column_name` in the arguments?
https://github.com/alquimistas-org/subida_cartera/pull/14/files#diff-92256a9f69e6bb74e01a73cf7fbace84472f5eea4f136547962e9992c59ac1b3R6

**Review by matiasbertani — COMMENTED** [review]
It looks amazing the new class created. Is cleaner this way to structure the code. I add some comments rewarding the new function of get_phones

File: tests/test_risk_data.py [hunk 2]
```diff
@@ -52,12 +60,12 @@ def test_read_osiris_account_return(self, mock_read_csv):
...
     @mock.patch("pandas.merge")
     @mock.patch("src.risk_data.read_osiris_accounts")
-    @mock.patch("src.risk_data.get_phone_risk")
+    @mock.patch("src.risk_data.get_phones")
     @mock.patch("src.risk_data.Escribir_Datos_Osiris")
     def test_risk_data(
         self,
         mock_Escribir_Datos_Osiris,
-        mock_get_phone_risk,
+        mock_get_phones,
```
[the remark this answers is not in the document]
Comment (natilou) [reply tests/test_risk_data.py:68]: It's a good idea, i will do it now. Thanks!

**Review by matiasbertani — APPROVED** [review]
lgtm

**Pull request merged**